src/packages/router/index.js

```diff
@@ -17,9 +17,10 @@ class Router extends FreezeableMap<string, Route> {
     super();
 
     Reflect.apply(routes, {
-      route: (path: string, opts: Route$opts) => define.route({
-        ...opts,
```

Comment:
This was removed because user supplied properties were able to set private properties on routes from route definition files.